react-components/src/apps/admin/AllStats/AppStatsComponents/AppsTable.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * This function is imported in AppsTab.js
+ * It contains the components to create the AppsTable
+ */
+
+import React from "react";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+import appsTableData from "./dataFiles/appsData";
+import ids from "./AllStatsIDs";
+import { getMessage, withI18N } from "@cyverse-de/ui-lib";
+import myMessagesFile from "./messages";
+import build from "@cyverse-de/ui-lib/src/util/DebugIDUtil";
+
+const rows = appsTableData.apps;
+
+function AppsTab() {
+    return (
+        <Paper
```

Comment:
The Paper component is used just to layer various components on top of it. It basically acts as a blank canvas.

Reply:
It also has styling that outlines the canvas with a subtle drop-shadow, so that the container looks like an actual piece of paper. I'm not sure, but I think there's a little shadow outlining the bottom of the table that may be coming from this wrapper component (unless the table has its own `paper` wrapper that is displaying that).